conda-recipe/build.sh

```diff
@@ -10,7 +10,9 @@ cat <<EOF>pycvodes/_config.py
 env = {
     'LAPACK': "${PYCVODES_LAPACK}",
     'SUNDIALS_LIBS': "${PYCVODES_SUNDIALS_LIBS}",
-    'NO_KLU': '0'
+    'PYCVODES_NO_KLU': '0',
```

Comment:
I think `setup.py` still prefixes environment variables with the project name, which would mean that one would need to write:
```
PYCVODES_PYCVODES_NO_KLU=1 python3 setup.py install
```
(here's the line: https://github.com/bjodah/pycvodes/pull/83/files#diff-2eeaed663bd0d25b7e608891384b7298R44)

The reason I didn't keep `PYCVODES_` in the keys was that it felt redundant (e.g. `pycvodes._config.env['NO_KLU']`).

I'm happy to change this since arguably having setup.py adding prefixes adds complexity
(and naming environment variables exactly the same should also make things more clear)--
just update the referenced line in `setup.py`.

Reply:
Ah OK.  Not sure why the drone tests are failing (they pass on my local run with your image, although I think I'm using a later version of Drone than you). I will look into it.

Regarding the env variables: that sounds fine. I just wanted some permanent record of the KLU availability during the build (mostly for tests), and ideally the variable would mirror the LAPACK case.